js/main.js

```diff
@@ -48,72 +49,104 @@ function extractResults(data) {
+  if (!resultsToDisplay) {
+    const targetElement = document.querySelector('.tag-line');
+    const closestParent = targetElement.closest('.row');
+    const noBooksContainer = document.createElement('div');
+
+    closestParent.insertAdjacentElement('afterend', noBooksContainer);
+    noBooksContainer.innerHTML = '<p>No Books Found! Search Again.</p>';
+  } else {
+    const searchContainer = document.querySelector('.search-results');
+
+    const searchResultsTitleContainer = document.createElement('div');
+    searchResultsTitleContainer.classList.add('search-results-title-container');
+
+    const searchResultsH2 = document.createElement('h2');
+    searchResultsH2.classList.add('search-results-title');
+    searchResultsH2.textContent = 'Search Results';
+
+    const searchListContainer = document.createElement('div');
+    searchListContainer.classList.add('search-list-container');
+
+    searchResultsTitleContainer.appendChild(searchResultsH2);
+    searchContainer.appendChild(searchResultsTitleContainer);
+    searchContainer.appendChild(searchListContainer);
+
+    resultsToDisplay.forEach((result) => {
+      const searchResultsContainer = document.querySelector('.search-list-container');
+
+      const searchResultsList = document.createElement('div');
+      searchResultsList.classList.add('search-results-list');
+
+      const bookDisplayInfo = document.createElement('div');
+      bookDisplayInfo.classList.add('book-display-info');
+
+      const bookImage = document.createElement('div');
+      bookImage.classList.add('book-image');
+
+      const bookImgPreview = document.createElement('img');
+      bookImgPreview.classList.add('book-img-preview');
+      bookImgPreview.src = result.previewLink;
+
+      const bookInformation = document.createElement('div');
+      bookInformation.classList.add('book-information');
+
+      const bookTitle = document.createElement('div');
+      bookTitle.classList.add('book-title');
+      bookTitle.textContent = result.title;
+
+      const bookAuthor = document.createElement('div');
+      bookAuthor.classList.add('book-author');
+      bookAuthor.textContent = result.author;
+
+      const bookNumPages = document.createElement('div');
+      bookNumPages.classList.add('book-num-pages');
+      bookNumPages.textContent = result.numPages;
+
+      const bookISBN = document.createElement('div');
+      bookISBN.classList.add('book-ISBN');
+      bookISBN.textContent = result.isbn;
+
+      const addBookBtnContainer = document.createElement('div');
+      addBookBtnContainer.classList.add('add-book-btn-container');
+
+      const addBookBtn = document.createElement('button');
+      addBookBtn.classList.add('add-book-btn');
+      addBookBtn.textContent = 'Add Book';
+
+      bookInformation.appendChild(bookTitle);
+      bookInformation.appendChild(bookAuthor);
+      bookInformation.appendChild(bookNumPages);
+      bookInformation.appendChild(bookISBN);
```

Comment:
You can use the append method to simplify your code: 
`bookInformation.append(bookTitle, bookAuthor, bookNumPages, bookISBN);
`